tools/topology/topology2/include/components/dcblock/default.conf

```diff
@@ -0,0 +1,12 @@
+# Exported Control Bytes 29-Aug-2023
```

Comment:
Can we comment on how these are generated i.e. so that users know what tool to run and any command line args used to generate this data (and it can be reproduced by them).

Reply:
Good that you noticed, this was generated with an earlier version of #8122. I need to update this new generated file that shows the text.

Reply:
It's fixed now